db/migrate/20170614135258_create_split_pdf_logs.rb

```diff
@@ -0,0 +1,22 @@
+class CreateSplitPdfLogs < ActiveRecord::Migration
+  def change
+    create_table :split_pdf_logs do |t|
+      t.string :split_pdf_console_log
```

Comment:
Remove this. We do not use standard out to capture logging or error information.